internal/functions/deploy/deploy.go

```diff
@@ -64,6 +64,14 @@ func Run(ctx context.Context, slugs []string, useDocker bool, noVerifyJWT *bool,
 	} else if err != nil {
 		return err
 	}
+
+	// Handle pruning if requested
+	if prune {
+		if err := pruneFunctions(ctx, slugs, flags.ProjectRef, force); err != nil {
```

Comment:
This is only considering auto-detected slugs and not those declared in config.toml